pombola/feedback/templates/feedback/add.html

```diff
@@ -33,43 +29,43 @@ <h1 class="page-title">
     </h1>
 
     <div class="infopage">
-    {% if submit_was_success %}
-
+    {% if message %}
         <div id="ajax_dialog_subcontent">
             <div class="success">
-                Thank you for your feedback!
+                {{ message|safe }}
             </div>
         </div>
+    {% else %}
```

Comment:
I don't think the logic
```
if message
else
    if submit was success
```
makes sense.

How about adding an else clause to `if submit_was_success` instead?

Reply:
It turns out `submit_was_success` is also used to determine whether it's a GET or a POST :facepalm: 

We're renaming `message` to be clearer. We should address `submit_was_success` some time as well.